src/cloud/project/project-upgrade.md

```diff
@@ -121,6 +121,7 @@ To check and update Zend Framework dependencies:
          "Magento\\Framework\\": "lib/internal/Magento/Framework/",
          "Magento\\Setup\\": "setup/src/Magento/Setup/",
          "Magento\\": "app/code/Magento/",
+         "Zend\\Mvc\\Controller\\": "setup/src/Zend/Mvc/Controller/",
    +     "Laminas\\Mvc\\Controller\\": "setup/src/Zend/Mvc/Controller/"
```

Comment:
so I am not sure if the updated version should have both lines, or only the Laminas line.

"Zend\\Mvc\\Controller\\": "setup/src/Zend/Mvc/Controller/",
"Laminas\\Mvc\\Controller\\": "setup/src/Zend/Mvc/Controller/"

@oshmyheliuk  Can you confirm?

In `composer.json` for 2.4 metapackage, only the first line is included: 

https://github.com/magento/magento-cloud/blob/879c6b42bfcb134536364563c9e83f5d74dc3309/composer.json#L33